packages/vkui/src/components/NativeSelect/NativeSelect.tsx

```diff
@@ -22,7 +22,7 @@ export interface NativeSelectProps
     HasRef<HTMLSelectElement>,
     HasRootRef<HTMLLabelElement>,
     HasAlign,
-    Pick<FormFieldProps, 'status'> {
+    Pick<FormFieldProps, 'before' | 'after' | 'status'> {
```

Comment:
Давай пока уберем эти свойства

```suggestion
    Pick<FormFieldProps, 'status'> {
```

Починю работу с ними отдельно

Reply:
убрал псевдо атрибуты , добавил placeholder?.length && styles['Select--hasPlaceholder'] где обнуляю падинг  , так ок?